src/no-enum.spec.ts

```diff
@@ -0,0 +1,156 @@
+// no-enum.spec.ts
+
+/*
+ * Copyright (c) 2021-2024 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+import rule, { ruleId } from './no-enum';
+import { RuleTester } from '@typescript-eslint/rule-tester';
+import { describe } from '@jest/globals';
+
+describe(ruleId, () => {
+  const ruleTester = new RuleTester();
+  ruleTester.run('no-side-effects', rule, {
+    valid: [
+      {
```

Comment:
i found it useful to provide 'name' property in each test case. it's similar to the name in Jest test cases, even though  it's optional.
it help when the test fails so that we know exactly where the error comes from.